assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java

```diff
@@ -1913,7 +1915,7 @@ public boolean canSubmit(Assignment a) {
     }
 
     @Override
-    @Transactional
+    // @Transactional(isolation = Isolation.SERIALIZABLE)
```

Comment:
is this commented out purposefully?

Reply:
yeah part of the clean up i will need to do